Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift

```diff
@@ -63,6 +63,9 @@ extension PhotonActionSheetProtocol {
         let copyAddressAction = PhotonActionSheetItem(title: Strings.CopyAddressTitle, iconString: "menu-Copy-Link") { _, _ in
             if let url = self.tabManager.selectedTab?.canonicalURL?.displayURL ?? urlBar.currentURL {
                 UIPasteboard.general.url = url
+                if let view = urlBar.window {
```

Comment:
What is the reason for this check?

Reply:
urlBar.window is type UIWindow? (Optional)

Reply:
![image](https://user-images.githubusercontent.com/2073238/104044941-d46b6e80-51ab-11eb-9059-f849b498c399.jpeg)

Reply:
Oh sorry, yes I see, thanks